components/popover/metadata/mods.md

```diff
@@ -1,18 +1,18 @@
-| Modifiable custom properties                  |
-| --------------------------------------------- |
-| `--mod-overlay-animation-duration`            |
-| `--mod-overlay-animation-duration-opened`     |
-| `--mod-popover-animation-distance`            |
-| `--mod-popover-background-color`              |
-| `--mod-popover-border-color`                  |
-| `--mod-popover-border-width`                  |
-| `--mod-popover-content-area-spacing-vertical` |
-| `--mod-popover-corner-radius`                 |
-| `--mod-popover-filter`                        |
-| `--mod-popover-pointer-edge-spacing`          |
-| `--mod-popover-pointer-height`                |
-| `--mod-popover-pointer-width`                 |
-| `--mod-popover-shadow-blur`                   |
-| `--mod-popover-shadow-color`                  |
-| `--mod-popover-shadow-horizontal`             |
-| `--mod-popover-shadow-vertical`               |
+| Modifiable custom properties              |
+| ----------------------------------------- |
+| `--mod-overlay-animation-duration`        |
+| `--mod-overlay-animation-duration-opened` |
+| `--mod-popover-animation-distance`        |
+| `--mod-popover-background-color`          |
+| `--mod-popover-border-color`              |
+| `--mod-popover-border-width`              |
+| `--mod-popover-content-area-spacing`      |
+| `--mod-popover-corner-radius`             |
+| `--mod-popover-dropshadow-blur`           |
+| `--mod-popover-dropshadow-color`          |
+| `--mod-popover-dropshadow-x`              |
+| `--mod-popover-dropshadow-y`              |
```

Comment:
Is this the correct way to write these properties? Should it be "dropshadow" or "drop-shadow"?